### ISSUE TITLE
Crash on load up -- Due to loot tables.

### ISSUE THREAD

**Issue by S7urp33** [S7urp33/ChronoEmu#16]
I loaded a fresh world db and applied all SQL updates and about every other time I load up the world it crashes upon reviewing the crash dump it goes back to loot. I will investigate our loot tables.

**Comment by Swater96** [comment]
if you're crashing on login and loading screen, I believe this is related to the guild update in player.cpp/player.h. Something is wrong with guild id.

Start with a fresh guilds table in the database and manually add a guild in the table and try to load the game. That resolved my crashing, see if it fixes yours

**Comment by Swater96** [comment]
Also appears to be a crash in worldsession, if you log out and log in 3 times quickly you will crash the server.

**Comment by S7urp33** [comment]
It's my warden stuff I left in I need to remove it there is a check

**Comment by S7urp33** [comment]
Did you see I moved all in front face checks to the DB and all values that were there in C++ I input into SQL

**Comment by S7urp33** [comment]
I will look into the guild ID issue and world session my last several crashes were due to my warden stuff I have applied locally so I did a fresh checkout in a separate folder

**Comment by Swater96** [comment]
Here is a debug run I did, I currently do not have any noticeable loot crashes but maybe this will help you look for the source.

```
 Error #16: UNADDRESSABLE ACCESS beyond heap bounds: reading 4 byte(s)
         LootMgr::LoadLootTables
             c:\users\sean\desktop\chronoemu\trunk\src\chrono-world\lootmgr.cpp(245):
         LootMgr::LoadLoot
             c:\users\sean\desktop\chronoemu\trunk\src\chrono-world\lootmgr.cpp(109):
         CallbackP0<>::execute 
```

**Comment by S7urp33** [comment]
Resolved in a3a39c433945ad5126c8ccc04557900df13ab4ae
